src/platform/ioc/container.ts

```diff
@@ -21,7 +21,13 @@ try {
  */
 @injectable()
 export class ServiceContainer implements IServiceContainer {
-    constructor(private container: Container) {}
+    public static get instance(): IServiceContainer {
```

Comment:
:thumbsup: I'd like to use the service container in some cases, instead of having all injected in the ctor.

Reply:
Leaving this for now, until we replace the DI conainer, thanks